gordo_components/builder/build_model.py

```diff
@@ -199,18 +154,16 @@ def _build(self) -> Tuple[Optional[sklearn.base.BaseEstimator], dict]:
         -------
             Tuple[Optional[sklearn.base.BaseEstimator], dict]
         """
```

Comment:
I know this is not part of the changes in this PR, but as a side note: Is the return signature here correct, and the model really optional? For me it seems like we are under the assumption that the model are going to exist, and that it will always return a "valid" model.

Reply:
I fix, I fix! :+1: